CardsCashCasino/Data/HoldEmUtil.cs

```diff
@@ -0,0 +1,335 @@
+﻿/*
+ *  Module Name:HoldEmUtil.cs
+ *  Purpose: Contains static utility methods used to determine the winner of a game of Texas Hold 'Em.
+ *  Inputs: The set of community cards and a player's hole cards.
+ *  Outputs: The ranking, optimal 5-card hand, and the value used to break ties.
+ *  Additional code sources: None
+ *  Developers: Ethan Berkley
+ *  Date: 11/08/2024
+ *  Last Modified: 11/08/2024
+ *  Preconditions: Hole card lists are of length 2, Community card lists are of length >= 3, the input to the tiebreaker function was returned by the GetScore function
+ *  Postconditions: None
+ *  Error/Exception conditions: Only if Preconditions were violated.
+ *  Side effects: None
+ *  Invariants: None
+ *  Known Faults: None encountered
+ */
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace CardsCashCasino.Data
+{
+    /// <summary>
+    /// Utility class that will be used to determine a hand's ranking and tie breaking score, which is used at the conclusion of a round.
+    /// Defines an enum and two functions:
+    /// The enum, Ranking, represents the Rank of the poker hand. i.e. Royal Flush, Four of a Kind, Full House, etc.
+    /// The first function, GetScore, takes in the community cards and the player's hole cards, and returns the optimal hand and it's ranking.
+    /// The second function, KickerValue, takes in a 5-card hand returned by GetScore, 
+    ///     and returns the value used to break ties between hands of the same rank.
+    /// </summary>
+    public class HoldEmUtil
```

Comment:
Rename this to PokerUtil. Most of the code will remain the same for getting a score for both HoldEm and Five Card Draw.